### PR TITLE
only show important branches (default, master, develop)

### PR DESCRIPTION
**Pull request by joshski**
So we can focus on important builds, this change hides any builds that aren't based on one of the following branches:

* `<default>` (where the build configuration does not cover multiple branches)
* `master`
* `develop`

/cc @slawomir-brzezinski-at-travcorp @nigeltaylor-ttc

### CODE REVIEW

File: Script/ViewModels/BuildScreenViewModel.js [hunk 1]
```diff
@@ -67,6 +67,8 @@
                         branchFromApi.buildType = buildType;
                         branchFromApi.builds = getBuildsForBranchObservable(branchFromApi);
                         return branchFromApi;
+                    }).filter(function(branchFromApi) {
+                        return ["<default>", "master", "develop"].indexOf(branchFromApi.name) > -1;
```
Comment (slawomir-brzezinski-at-travcorp) [inline Script/ViewModels/BuildScreenViewModel.js]: Thainks @joshski . 
No problem with this, except I would extract a naming convention like this to the Setting.js file, to make it easier to discover.
I'd simply move the whole function to Settings.js to a member like `Settings.branchFilter`

**Comment by joshski** [comment]
Happy @slawomir-brzezinski-at-travcorp ?

**Comment by slawomir-brzezinski-at-travcorp** [comment]
Hey @joshski . Actually, release branches are also important I think.

**Comment by joshski** [comment]
Sure. Do you know if they always follow a naming convention? Maybe we just look for the word "release"?

**Comment by slawomir-brzezinski-at-travcorp** [comment]
Yeah, at least for iTropics and Tropics this should work. In Tropics its always just 'release', in iTropics it's 'release/XX.XX'

**Pull request merged**